ledger/error/src/main/scala/com/daml/error/definitions/LedgerApiErrors.scala

```diff
@@ -1100,5 +1100,45 @@ object LedgerApiErrors extends LedgerApiErrorGroup {
       )(implicit loggingContext: ContextualizedErrorLogger)
           extends LoggingTransactionErrorImpl(cause = s"Inconsistent: $details")
     }
+
+    @Explanation("Errors that arise from an internal system misbehavior.")
+    object Internal extends ErrorGroup() {
+      @Explanation(
+        "The participant didn't detect an inconsistent key usage in the transaction." +
```

Comment:
I think you need a space after the dot in the string.

Reply:
Thanks, done 5d100450bb83920f3a5c50ba90f5a72e723008d6